telegram/update.py

```diff
@@ -23,50 +23,59 @@
 
 
 class Update(TelegramObject):
-    """This object represents a Telegram Update.
+    """
+    This object represents an incoming update.
 
-    Attributes:
-        update_id (int): The update's unique identifier.
-        message (:class:`telegram.Message`): New incoming message of any kind - text, photo,
-            sticker, etc.
-        edited_message (:class:`telegram.Message`): New version of a message that is known to the
-            bot and was edited
-        inline_query (:class:`telegram.InlineQuery`): New incoming inline query.
-        chosen_inline_result (:class:`telegram.ChosenInlineResult`): The result of an inline query
-            that was chosen by a user and sent to their chat partner.
-        callback_query (:class:`telegram.CallbackQuery`): New incoming callback query.
-        channel_post (Optional[:class:`telegram.Message`]): New incoming channel post of any kind -
-            text, photo, sticker, etc.
-        edited_channel_post (Optional[:class:`telegram.Message`]): New version of a channel post
-            that is known to the bot and was edited.
-        shipping_query (:class:`telegram.ShippingQuery`): New incoming shipping query.
-        pre_checkout_query (:class:`telegram.PreCheckoutQuery`): New incoming pre-checkout query.
+    Note:
+        At most one of the optional parameters can be present in any given update.
 
+    Attributes:
+        update_id (:obj:`int`): The update's unique identifier.
+        message (:class:`telegram.Message`): Optional. New incoming message.
+        edited_message (:class:`telegram.Message`): Optional. New version of a message.
+        channel_post (:class:`telegram.Message`): Optional. New incoming channel post.
+        edited_channel_post (:class:`telegram.Message`): Optional. New version of a channel post.
+        inline_query (:class:`telegram.InlineQuery`): Optional. New incoming inline query.
+        chosen_inline_result (:class:`telegram.ChosenInlineResult`): Optional. The result of an
+            inline query that was chosen by a user.
+        callback_query (:class:`telegram.CallbackQuery`): Optional. New incoming callback query.
+        shipping_query (:class:`telegram.ShippingQuery`): Optional. New incoming shipping query.
+        pre_checkout_query (:class:`telegram.PreCheckoutQuery`): Optional. New incoming
+            pre-checkout query.
```

Comment:
According to the styleguide document properties in their own methods. They show up as attributes in the docs.